File: pythonproblems/day2/palindrome.py
# check whether the given number is a palindrome or not
num=2322
num1=num
rev=0
while(num!=0):
    last=num%10
    rev=rev*10+last
    num=int(num/10)
if num1==rev:
    print(num1,"is a palindrome")
else:
    print(num1,"is a not a palindrome")

# check whether the given string  is a palindrome or not
str="mo M"
upd=str.replace(" ",'').lower()
reverse=upd[::-1]
if upd==reverse:
    print(str,"is a palindrome")
else:
    print(str,"is not")

File: jsprobelms/day5/palindrome.js
// check whether the given number is palindrome or not
num = 2322;
num1 = num;
rev = 0;
while (num != 0) {
  last = num % 10;
  rev = rev * 10 + last;
  num = Math.floor(num / 10);
}
// console.log(rev);

if (rev == num1) {
  console.log(num1, "is a palindrome");
} else {
  console.log(num1, "is not a palindrome");
}

// check whether the given string is palindrome or not
a = "mom";
rev1 = "";
for (i = a.length - 1; i >= 0; i--) {
  rev1 += a[i];
}
console.log(rev1);
if (a == rev1) {
  console.log(a, "is a palindrome");
} else {
  console.log(a, "is not a palindrome");
}
